AsyncSwift/Extensions/String+.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Any+.swift
+//  AsyncSwift
+//
+//  Created by Inho Choi on 2022/10/07.
+//
+
+import Foundation
+
+extension String {
+    func toStringArray() -> [String]? {
```

Comment:
```suggestion
    func convertToStringArray() -> [String]? {
```

method 이름은 동사로 시작되는게 좋아보입니다.

Reply:
수정했습니다!